src/elliptic/curves/bls12_381/g1.rs

```diff
@@ -684,16 +687,11 @@ mod tests {
         assert_eq!(b_tag, g);
     }
 
-    use pairing_plus::bls12_381::G1;
-    use pairing_plus::hash_to_curve::HashToCurve;
-    use pairing_plus::CurveProjective;
-    use pairing_plus::SubgroupCheck;
-
     #[test]
     fn base_point2_nothing_up_my_sleeve() {
-        let cs = &[1u8];
-        let msg = &[1u8];
-        let point = G1::hash_to_curve(msg, cs);
-        assert!(point.into_affine().in_subgroup());
+        use pairing_plus::SubgroupCheck;
```

Comment:
here and in G2 the purpose of the test is to prove that the second base point was generated honestly .
I got lazy but what should have been inside the method base_point2 is the result of this test (which optimise on not needing to compute the point each time we call it) (TODO)

Reply:
Just committed fixes. Now base_point2 returns precomputed point, and tests validate that point is precomputed honestly